### ISSUE TITLE
/usr/bin/env: "node\r":No such file or directory

### ISSUE THREAD

**Issue by jGleitz** [fritx/dwn#1]
When using the cli on Ubuntu, I get

```
$ node_modules/.bin/dwn my_address
/usr/bin/env: "node\r": No such file or directory
```

There seems to be an issue with line endings in this package.

**Comment by fritx** [comment]
Ugh, I had been testing,
but I guess I never tried to run the cli on system other than Windows though 😨 .

However, when I cloned the repo and was about to fix the problem,
it came out that my mac (OSX Yosemite 10.10.5) works fine with `\r` 😕 .
Never mind. ;)

I've published the fix under v0.0.2.
Please try it and let me know if it works.

@jGleitz good catch! Thanks!
I will take this lesson.

**Comment by jGleitz** [comment]
I can confirm this is fixed.
